net-libs/tox/tox-0_pre20160904.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit autotools eutils systemd user
+
+DESCRIPTION="Encrypted P2P, messaging, and audio/video calling platform"
+HOMEPAGE="https://tox.chat"
+EGIT_COMMIT="755f084e8720b349026c85afbad58954cb7ff1d4"
+SRC_URI="https://github.com/irungentoo/toxcore/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+av daemon log-debug log-error log-info log-trace log-warn +no-log ntox static-libs test"
+
+REQUIRED_USE="^^ ( no-log log-trace log-debug log-info log-warn log-error )"
+
+RDEPEND="
+	av? ( media-libs/libvpx
+		media-libs/opus )
+	daemon? ( dev-libs/libconfig )
+	ntox? ( sys-libs/ncurses )
+	>=dev-libs/libsodium-0.6.1[asm,urandom]"
+DEPEND="${RDEPEND}
+	test? ( dev-libs/check )
+	virtual/pkgconfig"
+
+S="${WORKDIR}/toxcore-${EGIT_COMMIT}"
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		$(usex log-trace "--enable-logging --with-log-level=TRACE" "") \
+		$(usex log-debug "--enable-logging --with-log-level=DEBUG" "") \
+		$(usex log-info "--enable-logging --with-log-level=INFO" "") \
+		$(usex log-warn "--enable-logging --with-log-level=WARNING" "") \
+		$(usex log-error "--enable-logging --with-log-level=ERROR" "") \
+		$(use_enable av) \
+		$(use_enable test tests) \
+		$(use_enable ntox) \
+		$(use_enable daemon) \
+		$(use_enable static-libs static)
+}
+
+src_install() {
+	default
+	if use daemon; then
+		newinitd "${FILESDIR}"/initd tox-dht-daemon
+		newconfd "${FILESDIR}"/confd tox-dht-daemon
+		insinto /etc
+		doins "${FILESDIR}"/tox-bootstrapd.conf
+		systemd_dounit "${FILESDIR}"/tox-bootstrapd.service
+	fi
+
+	prune_libtool_files
```

Comment:
We now prefer using find instead of `prune_libtool_files`, see: https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Handling_Libtool_Archives

Change it to the find command and then nix `eutils` from the inherit line